CampaignAutomation/Models/RLTBanner.swift

```diff
@@ -17,7 +17,7 @@ public struct RLTBanner: RLTDataModel, RLTLinkHandling {
     public let subtitle: String?
     public let imageUrl: URL?
     public let language: String?
-    internal let url: URL?
+    internal let url: String?
```

Comment:
@rossrossp  As Portal user can input any string to url field, app is unable to decode RLTBanner when url field is an invalid URL. So there's no items shown on the screen.
For example, when the response is a space string: `"url": " "`, the Banner items cannot be displayed on the screen.
Therefore, we should decode it as String type so that app can still display Banner items on the screen but do nothing when `generateLinkHandler` in `RLTLinkHandling` in this case. (line 26 in RLTLinkHandling.swift)
Note: Android also defines BannerDataModel's url as a `String?` type.

Reply:
OK thanks!